server/gendb/interface.go.tmpl

```diff
@@ -60,6 +60,12 @@ type Interface interface {
     Scan{{ $modelName }}sBy{{ $indexName }}(ctx context.Context, input Scan{{ $modelName }}sBy{{ $indexName }}Input, fn func(m *models.{{ $modelName }}, last{{ $modelName }} bool) bool) error
     {{- end }}
     {{- end }}
+    {{- range $transactWithSchemaName := $xdbConfig.EnableTransactions }}
+    {{- $transactWithModelName := pascalize $transactWithSchemaName }}
+    // TransactSave{{ $modelName }}And{{ $transactWithModelName }} saves {{ $modelName }} and {{ $transactWithModelName }} as an atomic transaction.
+    // Use the condition map to indicate pre-transaction condition requirements when updating an item; key = attribute JSON key, use 'x.y' notation for nested properties
```

Comment:
It would be nice if the condition map had type safety (and so maybe was just the actual model itself), but I think we'd lose the ability to distinguish nil values.